src/transformers/models/plbart/tokenization_plbart_fast.py

```diff
@@ -0,0 +1,271 @@
+# coding=utf-8
```

Comment:
Let's try to tackle this one in a final step once all the comments above are corrected